test/functional/tool_wallet.py

```diff
@@ -103,7 +103,7 @@ def test_tool_wallet_info(self):
             Transactions: 0
             Address Book: 1
         ''')
-        self.assert_tool_output(out, '-wallet=wallet.dat', 'info')
+        self.assert_tool_output(out, '-wallet=' + self.default_wallet_name, 'info')
```

Comment:
and this one is part of 20034, may be also rename it to "partial ... 20034"?

Reply:
bitcoin#20034 was supposed to be completed with [dash#4919](https://github.com/dashpay/dash/pull/4919), either through being overlooked or through OOO changes making previous behavior bleed in. 

because it's a very minor change relative to the whole PR, annotation seems unnecessary. omission was intentional.